kubectl-minio/cmd/tenant-delete.go

```diff
@@ -55,22 +54,27 @@ func newTenantDeleteCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 		Long:    deleteDesc,
 		Example: deleteExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := c.validate(); err != nil {
+			if err := c.validate(args); err != nil {
```

Comment:
I wonder if we should include a similar validation before running delete, since this is destructive (depending on PV reclaim policy, all stored data could be lost)